.github/workflows/build.yaml

```diff
@@ -26,7 +26,6 @@ jobs:
         # specified below, these parameters have no meaning on their own and
         # gain meaning on how job steps use them.
         include:
-          - python: 3.5
```

Comment:
Maybe `python: 3.9` could be added to the matrix? (as a "replacement" for 3.5)

Reply:
They already test for 3.9, I think it was your cut off in the preview here. :)

Reply:
Ah indeed!